lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -205,21 +197,20 @@ export default function NarrativeItinerariesHeader({
             <SortResultsDropdown
               id="sort-results"
               label={sortResultsLabel}
-              name={sortSelectedText}
+              name={sortText}
               title={sortResultsLabel}
             >
-              {sortOptions().map((x) => {
+              {sortOptions(intl).map((x) => {
                 return (
                   <li
                     key={x.value}
                     style={{
-                      fontWeight:
-                        sortSelectedText === x.text ? 'bold' : undefined
+                      fontWeight: sortText === x.text ? 'bold' : undefined
```

Comment:
Move this to CSS (use `button[aria-selected="true"]`) as with the locale selector.